Bottom: 0470bce436c7b350b0fc943f0ffcbb6dcbeda850
Top:    15175504b5fb703216a0ac4ac7cdd205dd69982f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 23:31:40 +0100

Refresh of double_click_opens_libs

---

diff --git a/src/hid/gtk/gui-library-window.c b/src/hid/gtk/gui-library-window.c
index c1f2ce7..4396d13 100644
--- a/src/hid/gtk/gui-library-window.c
+++ b/src/hid/gtk/gui-library-window.c
@@ -596,7 +596,7 @@ create_lib_treeview (GhidLibraryWindow * library_window)
   g_signal_connect (libtreeview,
                     "row-activated",
                     G_CALLBACK (tree_row_activated),
-                    compselect);
+                    NULL);
 
   /* connect callback to selection */
   selection = gtk_tree_view_get_selection (GTK_TREE_VIEW (libtreeview));
